stable/fluent-bit/values.yaml

```diff
@@ -181,7 +181,16 @@ input:
   tail:
     memBufLimit: 5MB
     path: /var/log/containers/*.log
-
+  systemd:
+    enabled: true
```

Comment:
This should be either off by default, or some upgrade note saying it will gather more logs then in the past. Some folks may have already configured systemd logging via rawConfig. We don't want to surprise them.

Reply:
Good point @kfox1111 ! Changed to disbaled by default